docs/catalog-api-use.md

```
@@ -0,0 +1,295 @@
+
+  // Delete catalog
+  await fetch('/cloudapi/1.0.0/catalogs/urn:vcloud:catalog:12345678-1234-1234-1234-123456789012', {
+    method: 'DELETE',
+    headers: {
+      'Authorization': `Bearer ${token}`
+    }
+  });
+
+  cURL Examples
+
+  # List catalogs
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)